components/Tabs/tests/unit/MDCItemBarTests.m

```diff
@@ -48,4 +75,50 @@ - (void)testItemAtIndexPath {
   XCTAssertNil([itemBar itemAtIndexPath:indexPathWithItemEqualToNegativeOne]);
 }
 
+- (void)testItemBarIsNotAccessibilityElement {
+  // Given
+  MDCItemBar *itemBar = [[MDCItemBar alloc] init];
+
+  // Then
+  XCTAssertFalse(itemBar.isAccessibilityElement);
+}
+
+- (void)testItemBarIsAccessibilityTabBarOniOS10Plus {
+  if (@available(iOS 10.0, *)) {
+    // When
+    self.itemBar.accessibilityTraits = UIAccessibilityTraitLink;
+
+    // Then
+    XCTAssertEqual(self.itemBar.accessibilityTraits,
+                   UIAccessibilityTraitTabBar | UIAccessibilityTraitLink);
+  }
+}
+
+- (void)testItemBarViewIncludesTabInAccessibilityLabelOniOS9 {
+  NSOperatingSystemVersion iOS10Version = {10, 0, 0};
+  if (![NSProcessInfo.processInfo isOperatingSystemAtLeastVersion:iOS10Version]) {
+    // When
+    UIView *view = [self.itemBar accessibilityElementForItem:self.itemBar.items.firstObject];
```

Comment:
Nit: Any reason not to iterate over all `itemBar.items`?

Reply:
No reason. Done.